package.json

```
@@ -16,119 +16,30 @@
     "node": "v20"
   },
   "packageManager": "npm@10.7.0",
+  "workspaces": [
+    "apps/*"
+  ],
   "scripts": {
-    "scripts:release-post": "cross-env NODE_NO_WARNINGS=1 node scripts/release-post/index.mjs",
-    "dev": "cross-env NODE_NO_WARNINGS=1 next dev --turbo",
-    "serve": "npm run dev",
-    "build": "cross-env NODE_NO_WARNINGS=1 next build",
-    "start": "cross-env NODE_NO_WARNINGS=1 next start",
-    "deploy": "cross-env NEXT_PUBLIC_STATIC_EXPORT=true npm run build",
-    "lint:js": "eslint \"**/*.{js,mjs,ts,tsx}\" --cache --cache-strategy=content --cache-location=.eslintjscache",
-    "lint:md": "eslint \"**/*.md?(x)\" --cache --cache-strategy=content --cache-location=.eslintmdcache",
-    "lint:css": "stylelint \"**/*.css\" --allow-empty-input --cache --cache-strategy=content --cache-location=.stylelintcache",
-    "lint": "turbo run lint:md lint:js lint:css",
-    "lint:fix": "turbo run lint:md lint:js lint:css --no-cache -- --fix",
+    "dev": "turbo run dev",
+    "build": "turbo run build",
+    "start": "turbo run start",
+    "check-types": "turbo check-types",
+    "lint": "turbo run lint",
+    "lint:fix": "turbo run lint --force -- --fix",
```

passing the argument another level down worked for me
```suggestion
    "lint:fix": "turbo run lint --force -- -- --fix",
```

---

Oops! Good find. Thank you, fixed that.

I had the same error on macOS and fixed it up now. Not sure if you want further platform validation.